src/Contracts/ChatWorkspacePromptsSettings.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Meilisearch\Contracts;
+
+class ChatWorkspacePromptsSettings extends Data
+{
+    public string $system;
+    public string $searchDescription;
+    public string $searchQParam;
+    public string $searchIndexUidParam;
```

Comment:
If you would pass empty string, will it still work? :)

Reply:
yes!

Reply:
But I guess it doesn't make sense. So it's best to encourage best practices.